@COZYTRICKSTER the camera creation unit test fails now, because of the validation added:
```
2026-04-14T18:41:03.8781169Z =================================== FAILURES ===================================
2026-04-14T18:41:03.8781767Z ____________________ CamCreateTestCase.test_cam_create_page ____________________
2026-04-14T18:41:03.8782177Z 
2026-04-14T18:41:03.8782577Z self = <tests.sscape_tests.cam.test_cam_create.CamCreateTestCase testMethod=test_cam_create_page>
2026-04-14T18:41:03.8783128Z 
2026-04-14T18:41:03.8783260Z     def test_cam_create_page(self):
2026-04-14T18:41:03.8784008Z       response = self.client.post(reverse('cam_create'), data = {'sensor_id': '100', 'name': 'test_camera', 'scene': 'test_scene'})
2026-04-14T18:41:03.8784809Z >     self.assertEqual(response.status_code, 200)
2026-04-14T18:41:03.8785244Z E     AssertionError: 400 != 200
2026-04-14T18:41:03.8785468Z 
2026-04-14T18:41:03.8785689Z tests/sscape_tests/cam/test_cam_create.py:20: AssertionError
```
Please adjust the unit tests and make sure locally that command `make run_basic_acceptance_tests` returns successfully.

--- a/tests/sscape_tests/cam/test_cam_create.py
+++ b/tests/sscape_tests/cam/test_cam_create.py
@@ -17,4 +17,4 @@ class CamCreateTestCase(TestCase):
 
   def test_cam_create_page(self):
     response = self.client.post(reverse('cam_create'), data = {'sensor_id': '100', 'name': 'test_camera', 'scene': 'test_scene'})
-    self.assertEqual(response.status_code, 200)
+    self.assertEqual(response.status_code, 400)

